### PR TITLE
Add support of H3ULCB ES2.0 Cetibox machine

### PR DESCRIPTION
**Pull request by iartemenko**

### CODE REVIEW

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg [hunk 1]
```diff
@@ -10,7 +10,7 @@ kernel = "/xt/domf/Image"
 
 extra = "root=/dev/xvda1 rw rootwait console=hvc0"
 
-memory = 192
+memory = 512
```
Comment (otyshchenko1) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg]: This change has nothing common with what this patch is intended to do. I think, this should be in a separate patch if there is a need.

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-kernel/linux/linux-generic-armv8.bbappend [hunk 2]
```diff
@@ -1,13 +1,15 @@
...
 BRANCH = "master"
+BRANCH_cetibox = "v4.14/rcar-3.7-ctc"
 SRCREV = "${AUTOREV}"
 LINUX_VERSION = "4.14.75"
 
 SRC_URI = " \
     git://github.com/xen-troops/linux.git;branch=${BRANCH} \
     file://defconfig \
   "
+
```
Comment (otyshchenko1) [inline recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-kernel/linux/linux-generic-armv8.bbappend]: [Here and everywhere down the code] Why this empty line was added?

File: recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/0001-sja1105-tool-avoid-GCC-8-warnings-treated-as-errors.patch [hunk 3]
```diff
@@ -0,0 +1,34 @@
+From fa0e88412c85739db154e1929588d012c8b4d9ff Mon Sep 17 00:00:00 2001
+From: xen-troops-buildbot <not.real.email@epam.com>
+Date: Tue, 26 Nov 2019 12:18:03 +0000
+Subject: [PATCH] PVRUM avoid GCC-8 warnings treated as errors
+
+Signed-off-by: xen-troops-buildbot <not.real.email@epam.com>
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/0001-sja1105-tool-avoid-GCC-8-warnings-treated-as-errors.patch]: This definitely should be updated with proper credentials

File: recipes-dom0/dom0-image-rescue-initramfs/files/meta-xt-prod-extra/recipes-kernel/linux/linux-generic-armv8.bbappend [hunk 4]
```diff
@@ -15,11 +15,19 @@ SRC_URI_append = " \
...
 
+###############################################################################
+# cetibox
+###############################################################################
+SRC_URI_append_cetibox = " \
+  file://r8a7795-h3ulcb-cetibox-dom0.dts;subdir=git/arch/${ARCH}/boot/dts/renesas \
+    file://r8a7795-h3ulcb-cetibox-domd.dts;subdir=git/arch/${ARCH}/boot/dts/renesas \  
+  file://defconfig \
+  "
+
```
Comment (otyshchenko1) [inline recipes-dom0/dom0-image-rescue-initramfs/files/meta-xt-prod-extra/recipes-kernel/linux/linux-generic-armv8.bbappend]: I don't quite understand why we need these added here

**Comment by otyshchenko1** [comment]
Also please update INSTALL file to mention about the machine in question.

File: recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/0001-sja1105-tool-avoid-GCC-8-warnings-treated-as-errors.patch (hunk 3 above)
Comment (otyshchenko1) [reply recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/0001-sja1105-tool-avoid-GCC-8-warnings-treated-as-errors.patch]: Also patch subject is not relevant (this is not a PVRUM)

File: recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/eth_full_reset.sh [hunk 5]
```diff
@@ -0,0 +1,21 @@
+#!/bin/sh
+GPIO_PATH=/sys/class/gpio
+
+# interface must be up for phytool to access PHYs
```
Comment (otyshchenko1) [inline recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/eth_full_reset.sh]: Don't we need this change here:
https://github.com/xen-troops/meta-xt-prod-aos/pull/35/commits/63acecb48c0c2d4f182338cfb3dddcc591a49bb8

Comment (andr2000) [reply recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/eth_full_reset.sh]: it seems we do

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg (hunk 1 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-extended/guest-addons/files/domf.cfg]: done, moved to a separate commit

File: recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-kernel/linux/linux-generic-armv8.bbappend (hunk 2 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-kernel/linux/linux-generic-armv8.bbappend]: fixed. with the only exception for readability

File: recipes-dom0/dom0-image-rescue-initramfs/files/meta-xt-prod-extra/recipes-kernel/linux/linux-generic-armv8.bbappend (hunk 4 above)
Comment (andr2000) [reply recipes-dom0/dom0-image-rescue-initramfs/files/meta-xt-prod-extra/recipes-kernel/linux/linux-generic-armv8.bbappend]: it was not added, but rather moved and grouped all together at one place

File: recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/0001-sja1105-tool-avoid-GCC-8-warnings-treated-as-errors.patch (hunk 3 above)
Comment (andr2000) [reply recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/0001-sja1105-tool-avoid-GCC-8-warnings-treated-as-errors.patch]: who's gonna be the author?

Comment (andr2000) [reply recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/0001-sja1105-tool-avoid-GCC-8-warnings-treated-as-errors.patch]: @iartemenko is the one

File: recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/eth_full_reset.sh (hunk 5 above)
Comment (andr2000) [reply recipes-domd/domd-image-minimal/files/meta-xt-prod-extra/recipes-connectivity/sja1105-tool/sja1105-tool/eth_full_reset.sh]: This is still not clear if it is needed or not. I would prefer we go without this patch in this PR

**Comment by andr2000** [comment]
Superseded by #50
